.husky/pre-commit

```diff
@@ -1,7 +1,13 @@
-#!/bin/sh
-SCRIPT_DIRECTORY=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+set -eu
+
+SCRIPT_DIRECTORY="$(dirname -- "$( readlink -f -- "$0"; )";)"
+
+echo "$SCRIPT_DIRECTORY"
```

Comment:
Do we need this or is this a debug/analysis remnant? ;)

Reply:
Thx! fixed